src/main/res/layout/loading_image_view.xml

```diff
@@ -0,0 +1,21 @@
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_horizontal">
+
+    <ImageView
+        android:id="@+id/product_imageView"
+        android:layout_width="128dp"
```

Comment:
why are the height and width hardcoded?  what is the value of using this if I need a different size?